kytos/core/link.py

```diff
@@ -171,18 +173,21 @@ def make_tags_available(
         tags: Union[int, list[int]],
```

Comment:
Can you update tags type `Union[int, list[int], list[list[int]]]` since you've augmented the underlying interface `make_tags_available` type too?

Reply:
Updated type.